09-intermediate-rails/creating-apis.md

```diff
@@ -166,7 +171,19 @@ def pet_params
 end
 ```
 
-As before, the big difference from a web app is in what we send back. While in a web app we would probably redirect the client to the resource they created, that doesn't make sense for an API. Instead, we just send back some JSON containing the ID of the newly created pet.
+As before, the big difference from a web app is in what we send back. While in a web app we would probably redirect the client to the resource they created, that doesn't make sense for an API. Instead, we just send back some JSON containing the ID of the newly created pet as well as the appropriate status code.
+
+**ALERT!** Rails is built with CSRF protection which essentially prevents malicious requests. You might remember seeing notes in passing about this in our forms curriculum, where we had to explicitly add an _authenticity token_ to each form.
+
+Since preventing these cross site requests doesn't apply in an API context, we will add a Rails helper method to our controller to allow these `POST` requests to go through:
```

Comment:
Should CSRF be its own section, with a full heading and everything? I can see how it fits here, but I think if I was a student trawling back through textbook pages trying to figure out why my capstone API doesn't work this would be hard to spot.

Reply:
Also, have you confirmed this is the only thing you need to do to get around CSRF?

Reply:
Sure I can separate it out into its own section - yes I ran through the example as I updated the notes